evm/src/cpu/kernel/constants/global_metadata.rs

```diff
@@ -103,6 +109,9 @@ impl GlobalMetadata {
             Self::BlockGasLimit,
             Self::BlockChainId,
             Self::BlockBaseFee,
+            Self::BlockGasUsed,
```

Comment:
What does `BlockGasUsed` represent? Looking in this PR, it doesn't seem to be used except for the check
```
builder.connect(
            x.block_metadata.block_gas_used,
            x.extra_block_data.gas_used_after,
);
```
which I don't really understand.

Reply:
Ah right, looks good then 👍